packages/astro-og-canvas/package.json

```
@@ -39,7 +39,7 @@
     "entities": "^4.4.0"
   },
   "peerDependencies": {
-    "astro": "^1.0.0 || ^2.0.0-beta"
+    "astro": "^1.0.0 || ^2.0.0-beta || ^3.0.0"
   },
   "engines": {
     "node": ">=16.0.0"
```

```suggestion
    "node": ">=18.14.1"
```

Astro now has a hard requirement on at least this node version: https://docs.astro.build/en/guides/upgrade-to/v3/#removed-support-for-node-16

---

I omitted that from this PR because these changes retain support for Astro 1 & 2, which do not have that version constraint and I believe that would cause false version constraint warnings or errors if someone who hasn't yet upgraded Astro upgrades this package. If a user of this package upgrades to Astro 3, they will pick up that stricter engine constraint from the [Astro package itself](https://github.com/withastro/astro/blob/main/package.json#L41-L42), which I believe should have the same effect.

---

Now that we're dropping support for Astro 1 & 2, I've applied this change